lib/beaker/dsl/helpers/puppet_helpers.rb

```diff
@@ -676,7 +676,10 @@ def stub_forge(forge_host = nil)
         def sleep_until_puppetdb_started(host, nonssl_port = nil, ssl_port = nil)
           nonssl_port = options[:puppetdb_port_nonssl] if nonssl_port.nil?
           ssl_port = options[:puppetdb_port_ssl] if ssl_port.nil?
-          curl_with_retries("start puppetdb", host, "http://localhost:#{nonssl_port}", 0, 120)
+          endpoint = 'status/v1/services/puppetdb-status'
+          retry_on(host,
+                   "curl -m 1 http://localhost:#{nonssl_port}/#{endpoint} | grep '\"state\":\"running\"'",
+                   {:max_retries => 120})
           curl_with_retries("start puppetdb (ssl)",
```

Comment:
I think that the `curl_with_retries` should be replaced same test as the `retry_on`—namely, it should also look for a status of running—but I'd like to standardize on how we attach certs to requests here, so it's not necessary for this PR.